src/ontology/cl-edit.owl

```diff
@@ -7800,43 +7800,51 @@ SubClassOf(obo:CL_0000528 obo:CL_0000540)
 AnnotationAssertion(rdfs:label obo:CL_0000529 "pigmented epithelial cell")
 SubClassOf(obo:CL_0000529 obo:CL_0000710)
 
-# Class: obo:CL_0000530 (primary neuron)
+# Class: obo:CL_0000530 (primary neuron (sensu Teleostei))
 
-AnnotationAssertion(rdfs:label obo:CL_0000530 "primary neuron")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:1842357") Annotation(oboInOwl:hasDbXref "doi:10.1242/dev.108.1.121") obo:IAO_0000115 obo:CL_0000530 "A neuron that develops during the early segmentation stages in teleosts, before the neural tube is formed.")
+AnnotationAssertion(rdfs:label obo:CL_0000530 "primary neuron (sensu Teleostei)")
 SubClassOf(obo:CL_0000530 obo:CL_0000540)
+SubClassOf(Annotation(obo:IAO_0000233 "https://github.com/obophenotype/cell-ontology/pull/1950") obo:CL_0000530 ObjectSomeValuesFrom(obo:RO_0002160 obo:NCBITaxon_32443))
 
-# Class: obo:CL_0000531 (primary sensory neuron)
+# Class: obo:CL_0000531 (primary sensory neuron (sensu Teleostei))
 
-AnnotationAssertion(rdfs:label obo:CL_0000531 "primary sensory neuron")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:1842357") obo:IAO_0000115 obo:CL_0000531 "A primary neuron (sensu Teleostei) that has a sensory function.")
```

Comment:
Did not find full article, only abstract. Did not find this cell type mentioned in abstract.

Reply:
I don’t have access to the full text either. The paper was suggested by @cerivs in the comments of #1931.